src/app.js

```diff
@@ -1 +1,42 @@
 'use strict';
+
+const minimist = require('minimist');
+
+const inputArray = minimist(process.argv.slice(2))._;
+
+const add = () => {
+  let total = 0;
+
+  for (let i = 1; i < inputArray.length; i++) {
+    total += inputArray[i];
+  }
+
+  return total;
+};
+
+const multiply = () => {
+  let total = 1;
+
+  if (inputArray.length < 2) {
+    return 'Nothing to multiply';
+  }
+
+  for (let i = 1; i < inputArray.length; i++) {
+    total *= inputArray[i];
+  }
+
+  return total;
+};
+
+const result = () => {
+  if (inputArray[0] === 'add') {
```

Comment:
i think it'd make sense to console.log `nothing to ${operation}` here
```suggestion
const result = () => {
	if (!args.length) { console.log(`There is nothing to ${operation}`); }
  if (operation === 'add') {
```